website/blog/2023-10-26-release-1.5.md

```diff
@@ -0,0 +1,219 @@
+
+Enhanced logic for displaying or hiding properties listed under the Settings > Preferences screens is now available. [#4159](https://github.com/containers/podman-desktop/pull/4159)
+
+<ReactPlayer playing controls url="https://user-images.githubusercontent.com/6422176/271650937-3991565c-12a4-4e6c-a315-9343bfa25f65.mov" width='100%' height='100%' />
+
+### Improved user experience for state changes
+
+The user experience around state changes for containers, pods, and other objects in the UI is improved, with clear status messages and improved animated visual indicator of state changes. [#4056](https://github.com/containers/podman-desktop/pull/4056)
+
+<ReactPlayer playing controls url="https://user-images.githubusercontent.com/19958075/270027524-f5176cf9-462f-4024-920a-b4a906c7d30d.mov" width='100%' height='100%' />
+
+<ReactPlayer playing controls url="https://user-images.githubusercontent.com/19958075/270027533-70e152ec-5bbf-45ad-9f1d-563752464655.mov" width='100%' height='100%' />
+
+### Extension API improvements
+
+The Podman Desktop extension API received many improvements, including:
+
+- In addition to pushing and listing image from an extension, it's now possible to pull images from a Podman Desktop extension [#4155](https://github.com/containers/podman-desktop/pull/4155)
+
+- The Podman Desktop extension API has been enhanced with both the ability to list images &amp; networks and the ability to create containers &amp; networks. [#4172](https://github.com/containers/podman-desktop/pull/4172)
+
+- Podman Desktop extensions now have a consistent way to run administrative tasks. [#4049](https://github.com/containers/podman-desktop/pull/4049)
+
+- Extensions now have the ability to register a custom Kubernetes config generator. [#3970](https://github.com/containers/podman-desktop/pull/3970)
+
+- The ability of extensions to add commands to UI menus has been extended; previously for action menus it was only available in the Image list screen. It is now possible for extensions to add commands to the action menus of items listed on the Containers list screen as well. [#3947](https://github.com/containers/podman-desktop/pull/3947) &amp; [#3963](https://github.com/containers/podman-desktop/pull/3963)
+
+- Extensions have gained the ability to contribute menu items in the UI based on specific conditions. [#3959](https://github.com/containers/podman-desktop/pull/3959)
+
+---
+
+## Other Notable Enhancements
+
+<!-- CATCHALL SECTION FOR MINOR ENHANCEMENTS -- >
+<!-- USE BULLET POINTS -->
+<!-- EXAMPLE -->
+<!-- - Show warning when creating a pod with two containers that use the same port [#2671](https://github.com/containers/podman-desktop/pull/2671). -->
+
+- The progress of loading an image into kind is now visible as a task in the Podman Desktop task manager. [#4061](https://github.com/containers/podman-desktop/pull/4061)
```

Comment:
Capitalize Kind. "as a task in the Podman Desktop task manager" is a bit redundant, maybe just "in the task manager".